src/app/components/board/board.component.ts

```diff
@@ -0,0 +1,94 @@
+import { Component, OnInit } from '@angular/core';
+import { ActivatedRoute, Router } from '@angular/router';
+import { Board } from 'src/app/constants/boardClass';
+import { BoardService } from 'src/app/services/board.service';
+import { ColumnService } from 'src/app/services/column.service';
+import { Column } from 'src/assets/interfaces/Column';
+
+@Component({
+  selector: 'app-board',
+  templateUrl: './board.component.html',
+  styleUrls: ['./board.component.scss'],
+  providers: [BoardService],
+})
+export class BoardComponent implements OnInit {
+  board!: Board;
+  showModal: boolean = false;
+  showCreateColumnModal: boolean = false;
+  showCreateTaskModal: boolean = false;
+  newColumnName: string = '';
+
+  constructor(
+    private route: ActivatedRoute,
+    private boardService: BoardService,
+    private columnService: ColumnService,
+    private router: Router
+  ) {}
+
+  ngOnInit() {
+    const boardId = this.route.snapshot.paramMap.get('_id');
+    localStorage.setItem('boardId', boardId ?? '');
```

Comment:
I would recommend moving logic related to localStorage to separate service. In case, the project migrate to some other storage you cat fix all necessary staff in one place